### ISSUE TITLE
Copy Transfer problem

### ISSUE THREAD

**Issue by permcody** [idaholab/moose#12434]
## Rationale

Possible bug in CopyTransfer

## Description

There appears to be an issue with CopyTransfer for a pretty simple case. This needs to be looked at.

Input files to reproduce are provided by this thread:
https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/moose-users/9Ps-Mmiylmk/ZrXR6t5UBAAJ

## Impact

Moderate: Transfer may not function properly

**Comment by starkeyk** [comment]
is there any update on this ?

**Comment by starkeyk** [comment]
MultiAppNearestNodeTransfer works for the same set up if this helps at all

**Comment by permcody** [comment]
Yeah, they are very different transfers. MultiAppNearestNode is a lot more
forgiving and works when meshes aren't aligned or not even on top of each
other. Copy is meant for speed. Nobody has looked into that specific issue.
We are all very busy right now. Glad you have found a workaround.
